pkgs/applications/networking/whalebird/default.nix

```diff
@@ -0,0 +1,53 @@
+{ appimageTools, fetchurl, lib, gsettings-desktop-schemas, gtk3
+, makeDesktopItem
+}:
+
+let
+  pname = "whalebird";
+  version = "4.3.0";
+
+  icon = fetchurl {
+    url = "https://raw.githubusercontent.com/h3poteto/whalebird-desktop/master/static/images/icon.png";
+    sha256 = "15i5jp1i0134acvh78f8x4dfdfm22sfr1j26cmih3haqlk0y694x";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = icon;
+    desktopName = pname;
+    genericName = "Whalebird fediverse client";
+    categories = "Application";
+  };
+
+in appimageTools.wrapType2 rec {
+  name = pname;
```

Comment:
```suggestion
  name = "${pname}-${version}";
```

Reply:
Done.

Reply:
Except this is a bad idea, because that is the name appimageTools uses for the program binary name. I'll revert this.